include/util.h

```diff
@@ -24,9 +25,11 @@
  */
 #define UL_CONST(x)   PASTE(x, lu)
 #define ULL_CONST(x)  PASTE(x, llu)
+#define NULL          ((void *)0)
```

Comment:
Does removing the void* not cause any warnings?

Not sure how you would tell unless you are compiling for all platforms.

Reply:
Hm, yes, it might, you're right.

I'll see if I can get at least the build part of the HW tests going so we can see this, but it would be good to not remove the cast in any case in the assembler version. @axel-h or is this something a specific assembler tripped up on for you?

Reply:
`NULL` is define as `0` if `__ASSEMBLER__` is defined, as the assembler does not know C types. Otherwise is remains as `(void*)0`. Thast allows having constants shares between C and assembly code to use NULL.